scripts/rename_resources.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import os
+import subprocess
+
+from azul.logging import (
+    configure_script_logging,
+)
+
+log = logging.getLogger(__name__)
+renamed = {
+    'aws_cloudwatch_log_group.azul_error_log': 'aws_cloudwatch_log_group.error_log',
+    'aws_cloudwatch_log_group.azul_index_log': 'aws_cloudwatch_log_group.index_log',
+    'aws_cloudwatch_log_group.azul_search_log': 'aws_cloudwatch_log_group.search_log',
+    'aws_cloudwatch_log_resource_policy.azul_es_cloudwatch_policy': 'aws_cloudwatch_log_resource_policy.index',
+    'aws_dynamodb_table.cart_items_table': 'aws_dynamodb_table.cart_items',
+    'aws_dynamodb_table.carts_table': 'aws_dynamodb_table.carts',
+    'aws_dynamodb_table.users_table': 'aws_dynamodb_table.users',
+    'aws_dynamodb_table.versions_table': 'aws_dynamodb_table.object_versions',
+    'aws_elasticsearch_domain.elasticsearch[0]': 'aws_elasticsearch_domain.index',
+    'aws_iam_role.state_machine_iam_role': 'aws_iam_role.states',
+    'aws_iam_role_policy.state_machine_iam_policy': 'aws_iam_role_policy.states',
+    'aws_s3_bucket.bucket': 'aws_s3_bucket.storage',
+    'aws_s3_bucket.url_bucket': 'aws_s3_bucket.urls',
+    'aws_sfn_state_machine.cart_export_state_machine': 'aws_sfn_state_machine.cart_export',
+    'aws_sfn_state_machine.cart_item_state_machine': 'aws_sfn_state_machine.cart_item',
+    'aws_sfn_state_machine.manifest_state_machine': 'aws_sfn_state_machine.manifest',
+    'null_resource.hmac-secret': 'null_resource.hmac_secret'
+}
+
+
+def main():
+    run_args = {
```

Comment:
If you defined a function called `terraform_state()` you might be able to reduce this further.